nise/generators/oci/oci_constants.py

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright 2023 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""OCI Report Constants"""
+from dataclasses import dataclass
+from dataclasses import field
+from random import choice
+
+from faker import Faker
+
+
+fake = Faker()
+
+
+@dataclass(frozen=True)
+class OCIReportConstantColumns:
+    """OCI report constant columns"""
+
+    tenant_id: str = field(default=f"ocid1.tenancy.oc1..{fake.pystr(min_chars=15, max_chars=25)}")
```

Comment:
These can be changed to type hinted declaration and assignment statements since only the `default` value is being used.
```suggestion
    tenant_id: str = f"ocid1.tenancy.oc1..{fake.pystr(min_chars=15, max_chars=25)}"
```